ui/resizable.js

```diff
@@ -1072,9 +1074,9 @@ $.ui.plugin.add("resizable", "ghost", {
 				margin: 0,
 				left: 0,
 				top: 0
-			})
-			.addClass("ui-resizable-ghost")
-			.addClass(typeof o.ghost === "string" ? o.ghost : "");
+			});
+		this._addClass( that.ghost, "ui-resizable-ghost",
+			typeof o.ghost === "string" ? o.ghost : "" );
```

Comment:
There is a class being added no matter what its just checking if there is an additional ghost class to add based on the ghost option.

HOWEVER...
that should be deprecated and this should just be the value for `ui-resizable-ghost`